packages/app/pages/course/[cid]/today.tsx

```diff
@@ -81,11 +99,14 @@ export default function Today(): ReactElement {
               <Title>Current Office Hours</Title>
               <TodayPageCheckinButton />
             </Row>
-            {role === Role.PROFESSOR && (
-              <Row>
-                <div>You are a professor for this course</div>
-              </Row>
-            )}
+            <Row>
+              <div>
+                <i>
+                  You are a <RoleColorSpan>{roleToString(role)}</RoleColorSpan>{" "}
+                  for this course
+                </i>
+              </div>
+            </Row>
             {course?.queues?.filter((q) => q.isOpen).length === 0 ? (
```

Comment:
I think we've talked about this before, but revisiting this it seems kinda weird that TAs can't see active Prof queues in the Queue tab + Today page... I know that TAs can't really do anything with a prof queue but it does still seem a bit straange maybe? @dfarooq610 curious for your thoughts

I think if we do wanna let TAs see prof queues, that will take another ticket as it requires modifying the GET /course endpoint